chrome-helper/eos-google-chrome.py

```diff
@@ -107,12 +107,27 @@ def _get_chrome_flatpak_launcher(self):
 
     parser = argparse.ArgumentParser()
     parser.add_argument('--debug', dest='debug', action='store_true')
+    parser.add_argument('--ozone-platform', dest='ozone_platform')
+    parser.add_argument('--ozone-platform-hint', dest='ozone_platform_hint')
```

Comment:
I think you should just set the hint default here instead of handling it manually below.

```suggestion
    parser.add_argument('--ozone-platform-hint', dest='ozone_platform_hint', default='auto')
```

Reply:
sure